src/js/osweb/items/generic_response.js

```diff
@@ -262,7 +261,7 @@ export default class GenericResponse extends Item {
       this.experiment.vars.acc = this.experiment.vars.accuracy
       this.experiment.vars.average_response_time = Math.round(this.experiment.vars.total_response_time / this.experiment.vars.total_responses)
       this.experiment.vars.avg_rt = this.experiment.vars.average_response_time
-      this.experiment.vars.set('correct_' + this.name, this.vars.correct)
+      this.experiment.vars.set('correct_' + this.name, this.experiment.vars.correct)

```

Comment:
I always though these were identical. Or rather, that if the variable wasn't found in the local var store, it automatically also looks in the global var store.

Reply:
This is how it works in an `inline_javascript` item, because there we use a proxy for the vars object. But for the regular `vars` object, only `vars.get()` uses this fallback behavior.